.gitignore

```diff
@@ -4,3 +4,5 @@ pkg/*
 .DS_Store
 Gemfile.lock
 coverage
+.history/
```

Comment:
I'd leave out editor specific configuration.
Personally, I'd recommend configuring [global gitignore](https://stackoverflow.com/a/7335487).

Reply:
thanks, you just saved me from some repetitive work :heart_eyes: